bucoffea/monojet/monojetProcessor.py

```diff
@@ -377,6 +379,16 @@ def process(self, df):
                 elif re.match(r'cr_g.*', region):
                     region_weights.add('trigger', np.ones(df.size))
 
+            if df['has_v_jet']:
+                if re.match(r'.*_loose_v.*', regions):
```

Comment:
this should be `region` rather than regions, same for the following lines

Reply:
That seems to be the bug I had, thanks!